app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
+    pass
+
+
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass
```

Comment:
Custom Exception Messages: 
It's a good practice to provide custom error messages in your custom exception classes. This helps in identifying the reason for an exception. Consider adding meaningful error messages to each exception class.

   Example:
   ```python
   class NotWearingMaskError(BaseException):
       def __init__(self, message="Visitor is not wearing a mask"):
           self.message = message
           super().__init__(self.message)